### DISCUSSION TITLE
1.4.0.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#267] [Обновления]
## Список изменений
- Исправлено появление "]" при возникновении ошибки

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.4.0...1.4.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.4.0.1'>1.4.0.1</a>.</em>

**Comment by NullVerdict** [comment]
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/052a88fc-f48b-4f0e-a9de-b639bee0b3b7)
